app/src/main/java/com/eveningoutpost/dexdrip/Home.java

```diff
@@ -425,7 +425,7 @@ private void displayCurrentInfo() {
     }
 
     @NonNull
-    private String extraStatusLine() {
+    public static String extraStatusLine(SharedPreferences prefs) {
```

Comment:
One is an instance/member variable the other one a parameter of a static function. No clash there.
But still: Android style would be calling the member `mPreferences`. And we could really use a bit more readable code. I'll change that tomorrow.